src/adcm_pytest_plugin/fixtures.py

```diff
@@ -139,7 +138,7 @@ def image(request, cmd_opts, adcm_api_credentials, additional_adcm_init_config):
     remove_docker_image(**init_image, dc=docker_client)
 
 
-def _adcm(image, cmd_opts, request, adcm_api_credentials, upgradable=False) -> Generator[ADCM, None, None]:
+def _adcm(image, cmd_opts, request, upgradable=False) -> Generator[ADCM, None, None]:
```

Comment:
Remove argument passing in the fixtures below

Reply:
Thanks, fixed